docs/reports/marin-32b-retro.md

```diff
@@ -0,0 +1,353 @@
+# Marin 32B Retrospective
+
+> Total tokens trained: ≈6.437T (Phase 1: 2.679T + Phase 3/QK‑Norm: 2.684T + Phase 4/Mantis cooldown: 1.074T; excludes diagnostic restarts and the abandoned Bison cooldown attempt).
+
+<!--
+
+- [Introduction](#introduction)
+- [Baseline Configuration](#baseline-configuration)
+  - [Architecture](#architecture)
+  - [Optimizer & Schedule](#optimizer--schedule)
+  - [Data Mix & Batch Schedule](#data-mix--batch-schedule)
+- [Training Phases](#training-phases)
+  - [Phase 1: Loss Spikes in `exp1295_32b`](#phase-1-loss-spikes-in-exp1295_32b)
+  - [Phase 2: Recovery Without Architecture Changes](#phase-2-recovery-without-architecture-changes)
+    - ["Necromancy" Restart (`exp1390_32b_necro`)](#necromancy-restart-exp1390_32b_necro)
+    - [Alternative Optimizers (`exp1388`, `exp1380`)](#alternative-optimizers-exp1388-exp1380)
+  - [Phase 3: Switch to QK-Norm (`exp1395_qwen3_32b`)](#phase-3-switch-to-qk-norm-exp1395_qwen3_32b)
+  - [Phase 4: Midtraining Runs](#phase-4-midtraining-runs)
+    - [Attempt 1 — Bison Cooldown (`exp1529_32b_bison_cooldown`)](#attempt-1--bison-cooldown-exp1529_32b_bison_cooldown)
+    - [Attempt 2 — Mantis Cooldown (`exp1529_32b_mantis_cooldown`)](#attempt-2--mantis-cooldown-exp1529_32b_mantis_cooldown)
+- [Lessons Learned](#lessons-learned)
+- [Glossary](#glossary) -->
```

Comment:
Can we link to the issue, PR, and data browser?  We need these for the front page as well